ruby.c

```diff
@@ -419,6 +426,11 @@ usage(const char *name, int help, int highlight, int columns)
     printf("%s""YJIT options:%s\n", sb, se);
     rb_yjit_show_usage(help, highlight, w, columns);
 #endif
+#if USE_ZJIT
```

Comment:
elif

Reply:
same discussion as https://github.com/ruby/ruby/pull/13197/files#r2066506891; this is deliberate.